CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright © 2010-2017 The CefSharp Authors. All rights reserved.
+// Copyright © 2010-2017 The CefSharp Authors. All rights reserved.
```

Comment:
Revert unnecessary formatting changes please.

Reply:
Sorry but my git client does not show me a change at this line (and as you see the two lines are identical). So I am not able to undo this line.

Reply:
Your editor has likely changed the files codepage.

The lines may look identical, if you did a binary diff they'd be different.

Reply:
should be fixed now